recipes/jtango/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "JTango" %}
+{% set version = "9.6.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tango-controls/JTango/releases/download/{{ version }}/JTango-{{ version }}.jar
+  sha256: bb80ed6c4c61160127a3c9255d6f74044202afe5d4112f627092afb584a68c94
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - openjdk >=8
+
+test:
+  commands:
+    - test -f ${PREFIX}/share/java/JTango.jar
+    - test -f ${PREFIX}/share/java/JTango-${PKG_VERSION}.jar
+
+about:
+  home: https://www.tango-controls.org
+  license: Apache-2.0, MIT, SMLNJ, EPL-1.0, BSD-2-Clause, BSD-3-Clause, LGPL-2.1-or-later, LGPL-3.0-or-later, MPL-2.0
```

Comment:
It should be sufficient to put the final license here and not all sublicenses, we mainly need the notices of the dependencies listed in the `license_file` section.

Reply:
OK, so to make sure I understand, the following should be enough?

```yaml
license: LGPL-3.0-or-later
license_file:
  - LICENSE
  - aggregate-third-party-report.html
```

Reply:
Yes